src/Card/index.tsx

```diff
@@ -4,20 +4,20 @@ import { default as AntCard, CardProps as AntCardProps, CardMetaProps as AntCard
 import './style.css'
 
 export type CardProps = AntCardProps & {
-  hasButton?:boolean,
-  hasMetaExtra?:boolean,
+  hasbutton?:string,
+  hasmetaextra?:string,
```

Comment:
Also the naming is incorrect. Should be camel case

Reply:
The reason i put the naming is because of it's showing the error of `"React does not recognize the `hasMetaExtra` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `hasmetaextra` instead. If you accidentally passed it from a parent component, remove it from the DOM element."`